packages/MorphicAPIExplorer-Tests.package/MEXLinter.class/instance/brokenVariableIsOnlyAssignedALiteralValue.st

```diff
@@ -0,0 +1,4 @@
+test-for-miscellaneous
+brokenVariableIsOnlyAssignedALiteralValue
+
```

Comment:
Missing comment why it is broken.

Reply:
Mit dem Test habe wir uns noch gar nicht mit auseinander gesetzt, der war schon vorher broken. Wir können uns das aber nochmal anschauen

Reply:
I see. Would be nice, but you don't need to as it's not THAT important :D